packages/react-reconciler/src/ReactFiberUnwindWork.js

```diff
@@ -407,9 +406,10 @@ function unwindWork(
         // Captured a suspense effect. Set the boundary's `alreadyCaptured`
         // state to true so we know to render the fallback.
         const current = workInProgress.alternate;
-        const currentState = current !== null ? current.memoizedState : null;
-        let nextState = workInProgress.memoizedState;
-        if (currentState === null) {
+        const currentState: SuspenseState | null =
+          current !== null ? current.memoizedState : null;
```

Comment:
I believe this will fix the issue in www, haven't confirmed yet. I'll try on my sandbox shortly.